doc/2/framework/classes/name-generator/generate-random-long-name/index.md

```diff
@@ -0,0 +1,24 @@
+---
+code: true
+type: page
+title: generateRandomLongName
+description: NameGenerator class generateRandomLongName() method
+---
+
+# generateRandomLongName
+
+<SinceBadge version="2.12.0" />
```

Comment:
We are generating the Kuzzle version where the feature will be landing by using this string
```suggestion
<SinceBadge version="auto-version" />
```

Reply:
Oh thanks for reminding, I forgot to change this and the other